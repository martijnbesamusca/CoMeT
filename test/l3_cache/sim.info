{'cmdline': ['../../run-sniper',
             '-v',
             '-s',
             'memTherm_core',
             '-n',
             '4',
             '-c',
             'gainestown_2_5D_l3_stacked',
             '--',
             './test',
             '-p',
             '4'],
 'git_revision': '70a94ea0a46a09195fc7cc97e7411ec90733f6a3',
 'host': '9f90132256d9',
 'pin_version': 'unknown',
 'rusage': (39.610402,
            0.7821159999999999,
            58548,
            0,
            0,
            0,
            56549,
            0,
            0,
            0,
            6592,
            0,
            0,
            0,
            250647,
            300),
 'snipercmd': ['bash',
               '-c',
               '/home/martijn/Documents/uva/thesis/CoMeT/lib/sniper -c /home/martijn/Documents/uva/thesis/CoMeT/config/base.cfg --general/total_cores=4 --general/output_dir=/home/martijn/Documents/uva/thesis/CoMeT/test/l3_cache --config=/home/martijn/Documents/uva/thesis/CoMeT/config/nehalem.cfg --config=/home/martijn/Documents/uva/thesis/CoMeT/config/gainestown_2_5D.cfg --config=/home/martijn/Documents/uva/thesis/CoMeT/config/gainestown_2_5D_l3_stacked.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/home/martijn/Documents/uva/thesis/CoMeT/test/l3_cache/sim.scripts.py -g --hooks/script0args= -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmp5FECeC/run_benchmarks'],
 't_elapsed': 44.257108211517334,
 't_start': 1655826580.122491,
 'tracecmds': [['/home/martijn/Documents/uva/thesis/CoMeT/record-trace',
                '-o',
                '/tmp/tmp5FECeC/run_benchmarks',
                '-v',
                '--pa',
                '-e',
                '1',
                '-s',
                '0',
                '-r',
                '1',
                '--follow',
                '--routine-tracing',
                '--',
                './test',
                '-p',
                '4']],
 'user': None,
 'vmem': 240308224L}
