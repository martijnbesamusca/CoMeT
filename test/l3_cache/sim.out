                                     | Core 0     | Core 1     | Core 2     | Core 3    
  Instructions                       |   40088076 |          0 |          0 |          0
  Cycles                             |   14642280 |   14642280 |   14642280 |   14642280
  IPC                                |       2.74 |       0.00 |       0.00 |       0.00
  Time (ns)                          |    4067300 |    4067300 |    4067300 |    4067300
  Idle time (ns)                     |         -1 |    4067300 |    4067300 |    4067300
  Idle time (%)                      |      -0.0% |     100.0% |     100.0% |     100.0%
Branch predictor stats               |            |            |            |           
  num correct                        |    4923346 |          0 |          0 |          0
  num incorrect                      |      52290 |          0 |          0 |          0
  misprediction rate                 |      1.05% |      0.00% |      0.00% |      0.00%
  mpki                               |       1.30 |       0.00 |       0.00 |       0.00
TLB Summary                          |            |            |            |           
  I-TLB                              |            |            |            |           
    num accesses                     |    7347185 |          0 |          0 |          0
    num misses                       |         46 |          0 |          0 |          0
    miss rate                        |      0.00% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.00 |       0.00 |       0.00 |       0.00
  D-TLB                              |            |            |            |           
    num accesses                     |   15216584 |          0 |          0 |          0
    num misses                       |       1481 |          0 |          0 |          0
    miss rate                        |      0.01% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.04 |       0.00 |       0.00 |       0.00
  L2 TLB                             |            |            |            |           
    num accesses                     |       1527 |          0 |          0 |          0
    num misses                       |        918 |          0 |          0 |          0
    miss rate                        |     60.12% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.02 |       0.00 |       0.00 |       0.00
Cache Summary                        |            |            |            |           
  Cache L1-I                         |            |            |            |           
    num cache accesses               |    7347185 |          0 |          0 |          0
    num cache misses                 |       1222 |          0 |          0 |          0
    miss rate                        |      0.02% |       inf% |       inf% |       inf%
    mpki                             |       0.03 |        inf |        inf |        inf
  Cache L1-D                         |            |            |            |           
    num cache accesses               |   15216584 |          0 |          0 |          0
    num cache misses                 |      66252 |          0 |          0 |          0
    miss rate                        |      0.44% |       inf% |       inf% |       inf%
    mpki                             |       1.65 |        inf |        inf |        inf
  Cache L2                           |            |            |            |           
    num cache accesses               |      68152 |          0 |          0 |          0
    num cache misses                 |      66008 |          0 |          0 |          0
    miss rate                        |     96.85% |       inf% |       inf% |       inf%
    mpki                             |       1.65 |        inf |        inf |        inf
  Cache L3                           |            |            |            |           
    num cache accesses               |      67068 |          0 |          0 |          0
    num cache misses                 |      54684 |          0 |          0 |          0
    miss rate                        |     81.54% |       inf% |       inf% |       inf%
    mpki                             |       1.36 |        inf |        inf |        inf
DRAM summary                         |            |            |            |           
  num dram accesses                  |     100288 |          0 |          0 |          0
  average dram access latency (ns)   |      63.97 |        inf |        inf |        inf
  average dram queueing delay        |      35.54 |       0.00 |       0.00 |       0.00
  average dram bandwidth utilization |     20.00% |      0.00% |      0.00% |      0.00%
Coherency Traffic                    |            |            |            |           
  num loads from dram                |      13041 |          0 |          0 |          0
  num loads from dram cache          |          0 |          0 |          0 |          0
  num loads from remote cache        |          0 |          0 |          0 |          0
